sys-apps/ndctl/ndctl-64.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools bash-completion-r1 linux-info systemd
+
+DESCRIPTION="Utility library for managing the libnvdimm sub-system in the Linux kernel"
+
+HOMEPAGE="http://pmem.io/"
+SRC_URI="https://github.com/pmem/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+# All the tests, like the rest of the nvdimm subsystem, require root to run
+# Futhermore, the tests require special testing kernel modules 
+RESTRICT="test"
+
+RDEPEND="dev-libs/json-c
+	sys-apps/keyutils
+	sys-apps/kmod
+	sys-apps/util-linux
```

Comment:
you do not need this here explicitly

Reply:
Are you talking about sys-apps/util-linux in RDEPEND? Do I still need it in DEPEND? It does link again libuuid which is part of sys-apps/util-linux - or do I need not need it because sys-apps/util-linux is aprt of the system set?

Reply:
Ok, after re-reading the ebuild guide I removed sys-apps/util-linux from the ebuild because it is a member of the system set and implicitly depended on.